resources/services/ec2/key_pairs.go

```diff
@@ -0,0 +1,78 @@
+package ec2
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/service/ec2"
+	"github.com/cloudquery/cq-provider-aws/client"
+	"github.com/cloudquery/cq-provider-sdk/provider/diag"
+	"github.com/cloudquery/cq-provider-sdk/provider/schema"
+)
+
+func Ec2KeyPairs() *schema.Table {
+	return &schema.Table{
+		Name:          "aws_ec2_key_pairs",
+		Description:   "Describes an EC2 Key Pair.",
+		Resolver:      fetchEc2KeyPairs,
+		Multiplex:     client.ServiceAccountRegionMultiplexer("ec2"),
+		IgnoreError:   client.IgnoreWithInvalidAction,
```

Comment:
Why did you choose this specific ignore function?
Will `client.client.IgnoreAccessDeniedServiceDisabled` work?

Reply:
I don't have a good reason for having chosen that specific ignore function. I will change it to client.IgnoreAccessDeniedServiceDisabled. If there is an even better ignore handler to use for this situation tell me and I will change it again.